geminimd/drive.py

```
@@ -0,0 +1,131 @@
+
+
+def find_folder_id(service, folder_name: str) -> Optional[str]:
+    cached = get_cached_folder_id(folder_name)
+    if cached:
+        try:
+            # Validate cached id exists
+            meta = service.files().get(fileId=cached, fields="id").execute()
+            if meta and meta.get("id"):
+                return cached
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)